src/rules/block-no-empty/index.js

```diff
@@ -27,7 +27,7 @@ export default function (actual) {
       let index = beforeBlockString(statement, { noRawBefore: true }).length
 
       // For empty blocks when using SugarSS parser
-      if (statement.raw("between") === undefined) {
+      if (statement.raws.between === undefined) {
```

Comment:
@davidtheclark not raw use stringify https://github.com/postcss/postcss/blob/979409b215bc70f5b69efe49a09dbc703642babe/lib/node.es6#L378 and return `stringify node` (in sugarss `a` -> `a{}`), i write about this above, we should always use `node.raws`, this return original content and we should set original offset(index)